crates/cli/src/model/datadog_api.rs

```diff
@@ -26,6 +27,26 @@ pub struct DiffAwareResponse {
     pub data: DiffAwareResponseData,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ConfigResponseDataAttributes {
+    #[serde(rename = "config_base64")]
+    pub config_base64: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ConfigResponseData {
+    #[serde(rename = "id")]
```

Comment:
Not asking you to remove, but as an FYI renames are not needed for field names that are the exact same as the json.